### PR TITLE
Owned change

### PR DESCRIPTION
**Pull request by iFA88**
ICO buy ether send changed to `require( beneficiaryAddress.call.gas(90000).value(0.2 ether)() );`
#114

### CODE REVIEW

File: ico.sol [hunk 1]
```diff
@@ -321,8 +257,8 @@ contract ico is safeMath {
         }
         uint256 _value = msg.value;
         if ( beneficiaryAddress.balance < 0.2 ether ) {
+            require( beneficiaryAddress.call.gas(90000).value(0.2 ether)() );
```
Comment (Dexaran) [inline ico.sol]: Why do we need 90000 gas for the fallback function?

Comment (iFA88) [inline ico.sol]: If anyone uses wallet contract, then 90k gas is optimal for use.

Comment (Dexaran) [inline ico.sol]: 90k gas will allow to call back the ICO contract recursively. Is it ok for you?

Comment (iFA88) [inline ico.sol]: Ofc. Because I don't like to feed a contract which recursive calls again the ICO contract.

**Comment by ItsmeeBosslady** [comment]
I'm new to GitHub but learning a lot already, thanks guys

**Pull request merged**